src/index.html

```diff
@@ -22,6 +22,376 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo image"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops &amp Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__nav-item">
```